kafka-sample-consumer-temperature-alert/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/segmentio/kafka-go"
+)
+
+type WeatherData struct {
+	StationId            int    `json:"stationId"`
+	StationName          string `json:"stationName"`
+	TemperatureInCelsius int    `json:"temperatureInCelsius"`
+}
+
+func loadPEMFromFile(filePath string) ([]byte, error) {
+	data, err := os.ReadFile(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read file %s: %w", filePath, err)
+	}
+	return data, nil
+}
+
+func main() {
+
+	serviceCert, err := loadPEMFromFile("/service.cert")
```

Comment:
```suggestion
	serviceCert, err := os.ReadFile("/service.cert")
```
Shall we use `os.ReadFile` directly as this doesn't do any additional transformation? Log entry alreay has sufficient info.